filetype/types/__init__.py

```diff
@@ -77,7 +81,24 @@
     archive.Ar(),
     archive.Z(),
     archive.Lz(),
+    archive.Rpm(),
+    archive.Elf(),
+    archive.Dcm(),
+    archive.Iso(),
 )
 
+# Supported application types
+APPLICATION = (
+    application.Wasm(),
+)
+
+# Supported application types
+DOCUMENT = (
+    document.Doc(),
```

Comment:
Doc type it will valid for xls and ppt because of same file signature, so no need to define the others.